ludwig/models/predictor.py

```diff
@@ -270,18 +271,21 @@ def __init__(self, model: ECD, gpus=None, gpu_memory_limit=None, allow_parallel_
         self.batch_evaluation = return_first(self.batch_evaluation)
 
 
-def calculate_overall_stats(output_features, predictions, dataset, training_set_metadata):
+def calculate_overall_stats(output_features, predictions, dataset, training_set_metadata, df_engine):
```

Comment:
At least in the docstring, we should call out that the "overall stats" that we are calculating is only on 10K rows.

This doesn't seem great though, as 10K seems like an arbitrary number.

Is there a way we could make this parameter configurable, like using an `eval_steps` parameter in the config (which parallels `train_steps`)?

This is along the lines of an API change that I proposed [here](https://docs.google.com/document/d/1T_CJqvnzH_TAA57-4U-E6vK5IjN8zHKoRHChNUGjR1I/edit#).

Reply:
I would consider this a stopgap for Dask until we have the full stats collection working. We could make it configurable, but ideally we should remove the need for this soon.